src/ToricVarieties/ToricDivisors/special_attributes.jl

```diff
@@ -0,0 +1,68 @@
+########################
+# 1: Special attributes of toric varieties
+########################
+
+@doc Markdown.doc"""
+    TrivialDivisor(v::AbstractNormalToricVariety)
+
+Construct the trivial divisor of a normal toric variety.
+For convenience, we also support `trivial_divisor(variety)`.
+
+# Examples
+```jldoctest
+julia> v = projective_space(NormalToricVariety, 2)
+A normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric variety without torusfactor
+
+julia> TrivialDivisor(v)
+A torus-invariant, non-prime divisor on a normal toric variety
+```
+"""
+@attr ToricDivisor function TrivialDivisor(v::AbstractNormalToricVariety)
+    return ToricDivisor(v, [fmpz(0) for i in 1:nrays(v)])
```

Comment:
You could also uses `zeros` (and elsewhere `ones`). I think it makes no difference performance wise, but perhaps is a bit more expressive.
```suggestion
    return ToricDivisor(v, zeros(fmpz, nrays(v)))
```

Reply:
Done. Thx Max!